### ISSUE TITLE
Manual errata: serial resonant bandpass filters component values

### ISSUE THREAD

**Issue by vk2tty** [profdc9/RFBitBanger#6]
TL;DR:
- 40m should read L2=0.58µH, L3=0.26µH not L2=0.77µH, L3=0.36µH
- 10m, 15m, 80m bands seem OK
- 80m C1=680pF mismatch batch 2 winding table C1=1000pF
- Remaining data might benefit from checking

G'day! I spotted a discrepancy in the band filter L2 and L3 columns while taking notes for a quick LCR check before soldering. Double-checking against [toroids.info](https://toroids.info/T50-6.php) for the 80m, 40m, 15m. and 10m bands found another typo. I haven't checked the others.

I also noticed the 80m C1 values didn't line up with the batch 2 builder's notes in OpenResearchInstitute/[RFBitBanger-kit](https://github.com/OpenResearchInstitute/RFBitBanger-kit) but don't know what to use as a tie-breaker.

While you're in there, tightening the C2+C3 column might let you widen L2 and L3 in turn making the table short enough to fit on the page with its header, which is currently widowed? Tiny nit-pick, I know.

73, VK2TTY

![RFBitBanger-Manual-p30-table-extract](https://github.com/profdc9/RFBitBanger/assets/151413186/d46e71d2-3466-4d9c-b4f4-3705452f0760)

**Comment by vk2tty** [comment]
G'day again! The manual's main PCB bill of materials gives 22µH for L6 and 100µH for L4.

The winding guide advises 12 turns on an FT37-43 for both, which toroids.info calculates as 50µH: half the manual's L6 inductance, and double the manual's L4 inductance. For what it's worth, both mine measured at 50µH with 12 turns.

Which of these figures should be corrected, and how?

**Comment by gweep** [comment]
Hi vk2tty,   My 80m and 40m filters work well as-built to the batch-2 instructions.  I did have to play with the winding spacing to get a nice filter response.   

My 20m filter (again built to batch-2 instructions) seems to be better suited for 17m with a peak around 18.1 MHz and a notch better suited for 20m (about 29MHz).   

Double checked my parts count and it appears I used all the correct parts.  

It looks like I am going to have to add turns to my 20m L2.   What is the source of your table above?  Not finding it in the Batch-2 docs.    

Does anyone have a link to the design equations for the RFBB low pass output filters?  I am not finding a suitable reference.  

Thanks,   (the slack message board is just a clutter of posts.  I find it hard to use.)

Mike
